src/pybind/mgr/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ if(WITH_MGR_ROOK_CLIENT)
 endif()
 if(WITH_TESTS)
   include(AddCephTest)
-  add_tox_test(mgr ${CMAKE_CURRENT_SOURCE_DIR} TOX_ENVS py3 py37 mypy flake8 jinjalint nooptional)
+  add_tox_test(mgr ${CMAKE_CURRENT_SOURCE_DIR} TOX_ENVS __tox_defaults__)
```

Comment:
for my clarity, what does `__tox_defaults__` actually resolve to here?

Reply:
It's a special sentinel value that's passed to the tox runner script. If the argument is equal to that string it just executes whatever the tox.ini file has configured in the envlist... e.g. https://github.com/ceph/ceph/blob/2d03e9841fe077c52d316becade5a01b2eb2987e/src/pybind/mgr/tox.ini#L2

Reply:
okay, does that mean we're running check-black and check-isort now as part of make check with this change?

Reply:
Correct. That's an intended part of this set of patches: to get those running as part of our regular test suite. Note that these environments only do anything for the smb module at this time, if other mgr modules need to be covered by these we need to update the tox.ini.